File: 01-Assignment-02-02-2026/Using_C/program_6.c
/* Program 6: Find circumference and area of a circle given radius */

#include <stdio.h>

int main() {
    // area = pi *r*r
    // circum = 2 *pi *r
    float area,circum,r;
    const float pi = 3.14159;

    printf("Enter Your circle's Radius : ");
    scanf("%f",&r);

    area = pi*r*r;
    circum = 2*pi*r;

    printf("Area of Circle : %.2f\n",area);
    printf("Circumference of Circle : %.2f",circum);
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program6.java
/*
Program 6
Java program to find the circumference and area of a circle with a given radius
*/

import java.util.Scanner;

public class Program6 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Enter the radius of circle : ");
        double radius = scanner.nextDouble();

        double circumference = 2 * 3.14 * radius;
        double area = 3.14 * radius * radius ;

        System.out.println("Circumference : " + circumference);
        System.out.println("Area : " + area);

        scanner.close();
    }

}
